Sources/InteroperabilityTests/InteroperabilityTestCases.swift

```diff
@@ -274,9 +274,18 @@ struct ServerStreaming: InteroperabilityTest {
 struct PingPong: InteroperabilityTest {
   func run(client: GRPCClient) async throws {
     let testServiceClient = Grpc_Testing_TestService.Client(client: client)
+    #if swift(>=5.9)
     let asyncStream = AsyncStream.makeStream(of: Int.self)
+    let continuation = asyncStream.continuation
+    let stream = asyncStream.stream
+    #else
+    let continuation: AsyncStream<Int>.Continuation!
+    let stream = AsyncStream(Int.self) { streamContinuation in
+      continuation = streamContinuation
+    }
+    #endif
```

Comment:
rather than doing this we can add a backported version of the API we want to this module: https://github.com/grpc/grpc-swift/blob/main/Sources/GRPCCore/Streaming/Internal/AsyncStream%2BMakeStream.swift